ifsa/src/main/java/nl/nn/adapterframework/webcontrol/pipes/TestIfsaService.java

```diff
@@ -181,7 +183,9 @@ private String processMessage(String applicationId, String serviceId,
 		sender.setMessageProtocol(messageProtocol);
 		sender.configure();
 		sender.open();
-		return sender.sendMessage("testmsg_" + Misc.createUUID(), message);
+		PipeLineSessionBase session = new PipeLineSessionBase();
+		session.put("id", "testmsg_" + Misc.createUUID());
```

Comment:
Wat is id? Waarom hebben we id, messageId, cid en tcid!?

Reply:
fixed